daemon/graphdriver/devmapper/driver.go

```diff
@@ -74,6 +74,7 @@ func (d *Driver) Status() [][2]string {
 		{"Metadata Space Used", fmt.Sprintf("%s", units.HumanSize(float64(s.Metadata.Used)))},
 		{"Metadata Space Total", fmt.Sprintf("%s", units.HumanSize(float64(s.Metadata.Total)))},
 		{"Metadata Space Available", fmt.Sprintf("%s", units.HumanSize(float64(s.Metadata.Available)))},
+		{"Thin Pool Minimum Free Space", fmt.Sprintf("%v", s.MinFreeSpace)},
```

Comment:
No, this is relevant for all thin pool. It does not matter if thin pool gets created out of loop devices or regular block devices or if it is an lvm thin pool.